docs/proposals/cri.md

```diff
@@ -0,0 +1,313 @@
+---
+title: CRI Design for edged
+authors:
+    - "@gpinaik"
+approvers:
+  - "@qizha"
+  - "@CindyXing"
+  - "@Baoqiang-Zhang"
+  - "@m1093782566"
+creation-date: 2019-04-19
+last-updated: 2019-04-21
+status: alpha
+---
+
+# CRI Support in edged
+
+* [CRI Support in edged](#CRI-Support-in-edged)
+  * [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non\-goals](#non-goals)
+  * [Proposal](#proposal)
+    * [Use Cases](#use-cases)
+  * [High Level Design](#high-level-design)  
+    * [Edged with CRI support](#edged-with-cri-support)
+  * [Low Level Design](#low-level-design)  
+    * [Packages](#packages)
+    * [Configuration parameters](#configuration-parameters)
+    * [Data structure modifications](#data-structure-modifications)
+    * [Edged object creation modifications](#edged-object-creation-modifications)
+    * [Runtime dependent module modifications](#runtime-dependent-module-modifications)
+    * [Runtime dependent functional modifications](#runtime-dependent-functional-modifications)
+  * [Open Questions](#open-questions) 
+    
+    
+## Motivation
+This proposal addresses the Container Runtime Interface support in edged to enable supporting multiple container
+runtimes like docker, containerd, crio etc on the edge platform.
```

Comment:
crio -> cri-o please.

Reply:
on the edge platform -> on the edge node.

Reply:
Addressed